README.md

```diff
@@ -14,63 +14,53 @@ Use these commmands to download, import data and create tiles jobs for Luxembour
 **If sudo required, use "-E"!!**
 
 ```bash
-docker-compose -f docker-compose.yml -f local-compose.yml down -v
+./exec.py shutdown
```

Comment:
Maybe we could make clearer that this command will delete existing volumes (imported data, generated tiles, etc.)   
`./exec.py clean` ?